views/blocks/payment_other.tpl

```diff
@@ -0,0 +1,50 @@
+<dl>
```

Comment:
I would call this template file `payment_with_logo` or similar instead as "other" is very generic.

Reply:
renamed to payment_other_with_logo.tpl to keep up with the Oxid template naming.